t/re-gmatch.t

```diff
@@ -0,0 +1,509 @@
+# vim:set ft= ts=4 sw=4 et fdm=marker:
+use lib 'lib';
+use Test::Nginx::Socket::Lua;
+use Cwd qw(cwd);
+
+#worker_connections(1014);
+#master_process_enabled(1);
+#log_level('warn');
+
+repeat_each(2);
+
+plan tests => repeat_each() * (blocks() * 4 - 5);
```

Comment:
It's not a good idea to substract a larger value (`5`) from the test block average (`4`) since it will lead to a negative test plan when running a test block individually via `--- ONLY`. Please change it to an addition calculation here instead.